contrib/containers/develop/Dockerfile

```diff
@@ -8,19 +8,34 @@ INCLUDE+ ci/Dockerfile
 # just started configuring this image, give us root access
 USER root
 
+# Make development environment more standalone
+RUN apt-get update && \
+    apt-get install $APT_ARGS \
+    apt-cacher-ng \
+    gdb \
+    gpg \
+    lsb-release \
+    nano \
+    openssh-client \
+    screen \
+    sudo \
+    zsh \
+    && \
+    rm -rf /var/lib/apt/lists/*
+
 # Discourage root access, this is an interactive instance
 #
 # Sometimes these commands are run repetitively _after_ the non-sudo
 # user was introduced and therefore these commands would fail
-# To mitigate the build halting, we've added "|| true" so that it
+# To mitigate the build halting, you can add "|| true" so that it
```

Comment:
Or `;:` to return true, two characters 😂

Reply:
We try to aim code to be self-documenting and readability usually helps in that regard but out of curiosity, will this work with most shells or is this a bash-ism?

Reply:
Yep, it is a bashism. The `;` is standard for any shell to chain next command without checking exit status of the previous, `:` generates a `0` exit code (true), while `!:` generates a `1` exit code (false).

Reply:
Interesting but I'd rather keep it more readable/cross-compatible.

Reply:
I agree that using `true` is more readable, I disagree we should still be trying to write POSIX compatible scripts in 2022, `bash` is ubiquitous in all *NIX platforms, you'll never move to a different shell.  Reasons to use the above syntax is for performance, one of the most expensive things you can do on Linux is create a new process and `true` and `false` are themselves processes, you should always prefer to use internal functions of the shell where ever possible to speed it up. That said, I'm not married to the idea, I just thought you might find it interesting. 😁

Reply:
It is quite interesting, I didn't consider the process spawn overhead and I'll be sure to keep this in mind. Thanks a lot for sharing this with us!